fdbserver/masterserver.actor.cpp

```diff
@@ -277,6 +284,7 @@ struct MasterData : NonCopyable, ReferenceCounted<MasterData> {
 		logger = traceCounters("MasterMetrics", dbgid, SERVER_KNOBS->WORKER_LOGGING_INTERVAL, &cc, "MasterMetrics");
 		if (forceRecovery && !myInterface.locality.dcId().present()) {
 			TraceEvent(SevError, "ForcedRecoveryRequiresDcID");
+			percentageOfTagsToBeSent = 110; // send random number of tags to the proxy
```

Comment:
Can you make this knob so that it's easier to test with different percentage numbers?

Reply:
Ok.

Reply:
Done.